### PR TITLE
Added timer device path as a CLI option

### PR DESCRIPTION
**Pull request by saramonteiro**
## Summary
This commit allows passing a timer device path argument to the timer example. 
## Impact
Now, the example may run with multiple timer instances.
## Testing
The change was tested by running the following command in the NuttX shell in the esp32 SoC which has 4 timers:
timer -d /dev/timerx (x = 0,1,2,3)

### CODE REVIEW

File: examples/timer/timer_main.c [hunk 1]
```diff
@@ -45,13 +45,15 @@
...
 #include <errno.h>
-
+#include <string.h>
 #include <nuttx/timers/timer.h>
 
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
 
+#define DEVNAME_SIZE 11
```
Comment (acassis) [inline examples/timer/timer_main.c:55]: Please change:
#define DEVNAME_SIZE 11 to 13
It is missing 1 byte to '\0' and also for some MCU with ten or more timers, /dev/timer10 will not fit

**Review by acassis — CHANGES_REQUESTED** [review]

**Comment by xiaoxiang781216** [comment]
@saramonteiro please don't forget to fix the nxsytle warning too:
https://github.com/apache/incubator-nuttx-apps/pull/404/checks?check_run_id=1162431000

File: examples/timer/timer_main.c (hunk 1 above)
Comment (saramonteiro) [reply examples/timer/timer_main.c]: done

**Comment by acassis** [comment]
LGTM

**Pull request merged**